dev-lang/dmd/dmd-2.067.1.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Systems programming language with C-like syntax & static typing"
+DESCRIPTION="The D programming language was started by Walter Bright, as a \
+'better C++' and has grown slowly, but steadily, for more than 15 years. Recent \
+improvements to the D development process have accelerated development of the \
+language and shown the need for a governing organization that overlooks and \
+arranges the future development of the D programming language."
+HOMEPAGE="https://dlang.org/"
+COPYRIGHT="1999-2014 Digital Mars"
+LICENSE="Digital Mars"
+REVISION="1"
+SOURCE_URI="https://github.com/dlang/dmd/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="526860456d181bb39ebdb7fb2bbb13e3afe62ec07c588eb5a5e8b8b61627ed8f"
+PATCHES="dmd-$portVersion.patchset"
+
+ARCHITECTURES="x86 x86_64"
+
+PROVIDES="
+	dmd$secondaryArchSuffix = $portVersion
+	cmd:dmd$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	dmd_data  == $portVersion
+	"
+
+if [ -z "$secondaryArchSuffix" ]; then
```

Comment:
You can probably remove this check if you don't intend to have support for secondary architectures.